app/models/conference.rb

```diff
@@ -6,6 +6,8 @@ class Conference < ActiveRecord::Base
   resourcify :roles, dependent: :delete_all
 
   default_scope { order('start_date DESC') }
+  scope :upcoming_conferences, (-> { where('end_date >= ?', Date.current) })
+  scope :past_conferences, (-> { where('end_date < ?', Date.current) })
```

Comment:
I would just call them `upcoming` and `past`, because `Conference.past_conferences` looks repetitive. `Conference.past` is pretty clear.